### PR TITLE
Revamp genetics-research text

### PR DESCRIPTION
**Pull request by hyanwong**

### CODE REVIEW

File: genetics-research.html [hunk 1]
```diff
@@ -13,12 +13,21 @@ <h1 class="hero__heading">
...
-                        file format is not only an efficient, scalable way to store and analyse genotype data, but as
-                        its underlying structure is a set of one or more trees it can be used to store and analyse phylogenetic data.
+                        Tskit enables rapid and principled genomic analysis. The tree
+                        sequence file format provides an optimised, scalable way to
+                        store genetic variation data, and underlies modern whole-genome
+                        simulation software such as
+                        <a href="software/msprime.html">msprime</a>. This is achieved
+                        by focussing on the evolutionary trees that underlie DNA
+                        sequence data, and providing efficent methods to analyse them.
+                        Tskit therefore provides a framework that serves to unify the
```
Comment (jeromekelleher) [inline genetics-research.html:23]: That's a big claim that might annoy people. How about "Tskit provides a common software framework that can be used across the fields of X Y and Z"?

File: genetics-research.html [hunk 2]
```diff
@@ -42,13 +51,26 @@ <h2 class="section__heading" id="popgen">
...
+                                    in a fraction of the space required by conventional
+                                    matrix-based methods.
+                                    Genetic variation data in tree sequence form
+                                    can be generated using a number of
+                                    simulation tools or inferred from existing data. 
+                                    
+                                    Once in tree sequence form, the tskit library
+                                    can be used for rapid analysis: there are
+                                    built-in functions that efficiently calculate many
+                                    conventional population genetic statistics. Because
```
Comment (jeromekelleher) [inline genetics-research.html:65]: s/conventional/standard perhaps?

File: genetics-research.html [hunk 3]
```diff
@@ -126,8 +148,31 @@ <h2 class="section__heading" id="statgen">
...
+                                    millions of multi-megabase genomes can take a few
+                                    seconds or less.
+                                    
+                                    <!--We can use trees to do things like efficient
+                                        matching via L/S HMMs (Duncan) or storing LD matrices
+                                        as graphical models (https://www.biorxiv.org/content/10.1101/2022.09.06.506858v1) -->
+                                    Storing genetic variation as tree or graph structures
+                                    allows further algorithmic efficiencies to be
+                                    exploited. This leads to scalable approaches to a
+                                    number of statisical genetic problems, for example
```
Comment (jeromekelleher) [inline genetics-research.html:167]: s/statisical/statistical

File: genetics-research.html [hunk 4]
```diff
@@ -126,8 +148,31 @@ <h2 class="section__heading" id="statgen">
...
+                                    matching genetic sequences against each other, and
+                                    when accounting for correlations along the genome.
```
Comment (jeromekelleher) [inline genetics-research.html:169]: delete the "when"?

File: genetics-research.html [hunk 5]
```diff
@@ -252,9 +297,37 @@ <h2 class="section__heading" id="phylogen">
...
+                                    with sample genomes and their ancestors in the tree.
+                                    <!-- Add in stuff about reading in trees, when we have it --> 
+
+                                    <!-- Other tree-based methods in tskit--> 
+                                    Tskit also provides generalizable methods to
+                                    calculate statistics based on tree branch lengths,
+                                    as well as topological methods that generate,
+                                    classify, and compare tree shapes. Optimised methods
+                                    are provided that map genetic variation
+                                    parsimoniously onto a tskit tree, and other
```
Comment (jeromekelleher) [inline genetics-research.html:321]: delete "tskit"?

File: genetics-research.html [hunk 6]
```diff
@@ -252,9 +297,37 @@ <h2 class="section__heading" id="phylogen">
...
+                                    efficient phylogenetic algorithms can be developed
+                                    in Python using the numba to achieve near-native speeds.
```
Comment (jeromekelleher) [inline genetics-research.html:323]:
```suggestion
                                    in Python using numba to achieve near-native speeds.
```

Maybe make this a link to the numba webpage?

File: genetics-research.html [hunk 7]
```diff
@@ -252,9 +297,37 @@ <h2 class="section__heading" id="phylogen">
...
+                                    
+                                    <!-- Easy extension to recombinant genealogies --> 
+                                    Unlike many other phylogenetic libraries, tree
```
Comment (jeromekelleher) [inline genetics-research.html:326]:
```suggestion
                                    Unlike other phylogenetic libraries, tree
```
there's no other library that takes recombination into account, right?

File: genetics-research.html [hunk 8]
```diff
@@ -331,15 +404,18 @@ <h2 class="section__heading" id="developers">
...
-                                    Examples of using the tskit C API can be found
-                                    <a href="https://github.com/tskit-dev/tskit/tree/main/c/examples">here</a>
-                                    and extensive Python API usage is found in the
-                                    <a href="https://tskit.dev/tutorials/tables_and_editing.html">tskit tutorials</a>.
+                                    Tskit is a lightweight library that provides both a
+                                    Python and C API for reading, writing
+                                    and manipulating succinct tree sequences. These APIs
+                                    can be used by simulation and inference tools to
+                                    write tree sequences to disk, and by analysis tools
+                                    to read them, decode their genotypes, perform efficient
```
Comment (jeromekelleher) [inline genetics-research.html:412]:
```suggestion
                                    to read them, decode their nucleotide sequences, perform efficient
```

"decode genotypes" might confuse?

**Review by jeromekelleher — APPROVED** [review]
Looks great, thanks! some suggestions

File: genetics-research.html (hunk 1 above)
Comment (hyanwong) [reply genetics-research.html:23]: Good point. That's a nice phrasing

File: genetics-research.html (hunk 3 above)
Comment (hyanwong) [reply genetics-research.html:167]: Good spot.

File: genetics-research.html (hunk 6 above)
Comment (hyanwong) [reply genetics-research.html:323]: Yes, there are a number(!) of places where we could add links. I'll do this one, and we can make a second pass for the others.

**Pull request merged**